include/deal.II/physics/transformations.h

```diff
@@ -29,6 +30,73 @@ namespace Physics
   {
 
     /**
+     * Transformation functions and tensors that are defined in terms of
+     * rotation angles and axes of rotation.
+     *
+     * @author Jean-Paul Pelteret, 2017
+    */
+    namespace Angular
+    {
+      /**
+       * @name Rotation matrices
+       */
+//@{
+
+      /**
+      * Returns the rotation matrix for 2-d Euclidean space, namely
+      * @f[
+      *  \mathbf{R} := \left[ \begin{array}{cc}
+      *  cos(\theta) & sin(\theta) \\
+      *  -sin(\theta) & cos(\theta)
+      * \end{array}\right]
+      * @f]
+      * where $\theta$ is the rotation angle given in radians.
+      * In particular, this describes the rotation of a vector
+      * relative to a <a href="http://mathworld.wolfram.com/RotationMatrix.html">fixed set of axes</a>.
+      *
+      * @param[in] angle The rotation angle (about the z-axis) in radians
```

Comment:
I've updated both of the descriptions of the rotations to be more specific. Would you mind please confirming that you're happy with the changes I've made?